docs/content/basic/practices.rst

```diff
@@ -8,34 +8,30 @@ Practices
 
 Here we cover
 
-- :ref:`Testing <testing>`
-- :ref:`Style <style>`
-- :ref:`Licensing <licensing>`
+- `testing`_
+- style_
+- licensing_
```

Comment:
Are there intentionally backticks by testing, but not by style nor by licensing?